src/multisig.rs

```diff
@@ -182,10 +193,10 @@ impl MultiSig {
         Ok(address)
     }
 
-    pub fn partially_sign(&self, public_keys: &[PublicKey], secret: &RandomSecret, commitments: &[Commitment], data: &[u8]) -> PartialSignature {
-        let kp = KeyPair::from(self.private_key.clone());
-        kp.partial_sign(public_keys, secret, commitments, data).0
-    }
+    // pub fn partially_sign(&self, public_keys: &[PublicKey], secret: &RandomSecret, commitments: &[Commitment], data: &[u8]) -> PartialSignature {
+    //     let kp = KeyPair::from(self.private_key.clone());
+    //     kp.partial_sign(public_keys, secret, commitments, data).0
+    // }
```

Comment:
I would move parts of the transaction.rs file to here. Logically it makes sense to still have a partially_sign function.